src/Swarm/TUI/Model.hs

```diff
@@ -378,6 +382,12 @@ uiModal :: Lens' UIState (Maybe Modal)
 -- | A togle to show the FPS by pressing `f`
 uiShowFPS :: Lens' UIState Bool
 
+-- | A togle to show inventoy items whith count 0 by pressing `0`
+uiShowZero :: Lens' UIState Bool
+
+-- |
+uiInventoryShouldUpdate :: Lens' UIState Bool
```

Comment:
Missing comment :wink:

Reply:
What do you think I should change it to?

Reply:
Probably just mention what toggles it (and that it is used when viewing same robots inventory?), also see my next comment.